### PR TITLE
Module split

### PR DESCRIPTION
**Pull request by codepitbull**
This PR adds two new modules, one containing the old Scala driver, the other one the new Kotlin based driver by @oshai. This should start the transition discussewd in #125

### CODE REVIEW

File: vertx-mysql-postgresql-client-jasync/README.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+= Vert.x MySQL PostgreSQL client
+
+image:https://travis-ci.org/vert-x3/vertx-mysql-postgresql-client.svg?branch=master["Build Status", link="https://travis-ci.org/vert-x3/vertx-mysql-postgresql-client"]
+
+This Vert.x client uses the https://github.com/mauricio/postgresql-async[Mauricio Linhares driver] to support
```
Comment (oshai) [inline vertx-mysql-postgresql-client-jasync/README.adoc]: This should probably be changed. not sure if you want to do that as part of this commit.

**Review by oshai — APPROVED** [review]

File: vertx-mysql-postgresql-client-jasync/README.adoc (hunk 1 above)
Comment (codepitbull) [reply vertx-mysql-postgresql-client-jasync/README.adoc]: Fixed

Comment (oshai) [reply vertx-mysql-postgresql-client-jasync/README.adoc]: Thanks

**Comment by okou19900722** [comment]
both modules used  `jasync`. none with `mauricio`

**Comment by codepitbull** [comment]
Holy crap, something went wrong.
Will fix asap.

**Comment by codepitbull** [comment]
@okou19900722  thanks for pointing out, it's fixed now.

**Comment by okou19900722** [comment]
great

**Pull request merged**